gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Dec 12 15:49:37 IST 2017
+#Mon Apr 30 11:20:13 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
Unneeded

Reply:
@iamareebjamal Builds fail without this update.. It won't be able to pass CircleCI tests. Please suggest, what should be done?

![image](https://user-images.githubusercontent.com/30979369/40247715-f3aebd34-5aea-11e8-93d5-a9cdb40f408e.png)

Reply:
Then a PR updating dependencies should be merged before this. This cannot be added in this PR

Reply:
Sure.  I will do this asap.

Reply:
@iamareebjamal As all dependencies have been updated now, this PR too has passed all checks. It would be great if you could please review this too.